src/GraphQL/loaders/defaultGraphQLTypes.js

```diff
@@ -964,8 +921,6 @@ const STRING_WHERE_INPUT = new GraphQLInputObjectType({
     in: inOp(GraphQLString),
     notIn: notIn(GraphQLString),
     exists,
-    inQueryKey,
```

Comment:
Hummm, from my point of view it's hard to understand this query without reading a documentation.  The `inQueryKey` feature is interesting to implement but i think we need first to re-design the input before re-implementing this feature. 😄  ? like playing with `Enum` for `key` and `className`, but this is not the purpose of this PR... We need to avoid people using inputs that can be deprecated in a near futur 😄

Reply:
So I think it would be better keep it here while we don't improve. Someone can be already using this feature and they will have no alternative.